test/Lint.lean

```diff
@@ -210,24 +276,6 @@ example : ℕ → ℕ := set_option linter.style.lambdaSyntax true in λ _ ↦ 0
 
 end lambdaSyntaxLinter
 
-set_option linter.globalAttributeIn false in
```

Comment:
This fixes a merge error: the same test is already present on line 158.

Reply:
Indeed, although its [line 127 on the resulting file](https://github.com/leanprover-community/mathlib4/blob/3c4d23a560483d82fe8ad39580429fcfb3bc5936/test/Lint.lean#L127-L143).